frontend/src/metabase-lib/Question.ts

```diff
@@ -497,18 +497,30 @@ class Question {
     }
   }
 
+  /**
+   * The name is somewhat misleading because this method applies not only
+   * for models (datasets) but also for metrics. When opening either one,
```

Comment:
1. Open up a metric
2. Select "Duplicate" from "..." menu
3. Duplicate it
4. Answer "Not now" when asked about adding to a dashboard
5. Click big blue play button

:x: URL changes from `/metric/:id-slug` to `/metric/:id-slug#base64`

_(perhaps it's the same case as previous one)_